InscryptionAPI/Card/TribeManager.cs

```diff
@@ -47,7 +47,7 @@ public static void UpdateTribeIcon(CardDisplayer3D __instance, CardInfo info)
         [HarmonyPostfix]
         public static void GetCardbackTexture(ref Texture __result, CardChoice choice)
         {
-            if (__result == null && choice.resourceType != ResourceType.Blood && choice.resourceType != ResourceType.Bone)
+            if (__result == null && choice.resourceType == ResourceType.None)
```

Comment:
Should this just be 

```c#
choice.tribe != Tribe.None
```

Reply:
I feel like making sure the result is null is a safe option to check for. Tho yeah, instead of `choice.resourceType == ResourceType.None` you could do `choice.tribe != Tribe.None`